CMakeLists.txt

```diff
@@ -456,6 +464,25 @@ if(WITH_CORE)
   find_package(EXIV2 REQUIRED)
 
   # optional
+  if (WITH_SFCGAL)
+
+    if(POLICY CMP0167) # avoid deprecation warning from CGAL
+      cmake_policy(SET CMP0167 NEW)
+    endif()
+
+    find_package(SFCGAL) # SFCGAL provider
+    message(STATUS "Found SFCGAL: ${SFCGAL_VERSION} ${SFCGAL_DIR}")
+    if(SFCGAL_VERSION VERSION_LESS "2.1")
```

Comment:
Please lower this to 2.0 -- 2.1 is much too recent (it isn't even available on Fedora 42)

```suggestion
    if(SFCGAL_VERSION VERSION_LESS "2.0")
```

Reply:
> We need at least 2.1 and it is available in rawhide

Can you elaborate why? Rawhide isn't a realistic option.

Reply:
Right, but none of those improvements should force 2.1. I think the minimum should be conservative (ie 2.0) with newer features conditionally compiled in. Exactly like what we do for Geos library features.

Reply:
@ptitjano @lbartoletti what do you think?

Reply:
I have updated the PR. The min version is now SFCGAL 2.0